projects/plugins/jetpack/class.jetpack-cli.php

```diff
@@ -92,7 +92,11 @@ public function status( $args ) {
 		} else {
 			$error = array();
 			foreach ( $cxntests->list_fails() as $fail ) {
-				$error[] = $fail['name'] . ': ' . $fail['message'];
+				$error[] = sprintf(
+					'%1$s%2$s',
+					$fail['name'],
+					! empty( $fail['message'] ) ? ': ' . $fail['message'] : ''
+				);
```

Comment:
Could go a little simpler if you want
```suggestion
				$error[] = $fail['name'] . ( empty( $fail['message'] ) ? '' : ': ' . $fail['message'] );
```

Reply:
Works for me. I found `sprintf` to be more readable, but happy with that too. Committing.

Reply:
For the record, I didn't have any real problem with the sprintf (which is why I approved), it just struck me as a little more complicated so I offered the alternative in case you liked it.